File: ProAgil.Domain/Speaker.cs
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string MiniCurriculum { get; set; }
        public string ImageUrl { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }

        public List<SocialNetwork> SocialNetwork { get; set; }
        public List<EventSpeaker> EventSpeakers { get; set; }
    }
}

File: ProAgil-App/src/app/models/Speaker.ts
import { SocialNetwork } from './Socialnetwork';
import { Event } from './Event';
import { BaseResourceModel } from './base-resource.model';

export class Speaker extends BaseResourceModel {
  constructor(
    id: number,
    name: string,
    miniCurriculum: string,
    imageUrl: string,
    phone: string,
    email: string,

    socialNetwork: SocialNetwork[],
    eventSpeakers: Event[]
  ) {
    super();
  }
}
